--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -5270,12 +5270,8 @@ test("unit_tests") {
       "//chrome/app_shim",
       "//chrome/browser/apps/app_shim",
       "//chrome/common/notifications",
-      "//chrome/common/safe_browsing:archive_analyzer_results",
-      "//chrome/common/safe_browsing:disk_image_type_sniffer_mac",
       "//chrome/services/mac_notifications:unit_tests",
       "//chrome/services/mac_notifications/public/mojom",
-      "//chrome/utility/safe_browsing/mac",
-      "//chrome/utility/safe_browsing/mac:dmg_common",
       "//ui/events/devices:test_support",
     ]
   }
--- a/chrome/utility/safe_browsing/mac/BUILD.gn
+++ b/chrome/utility/safe_browsing/mac/BUILD.gn
@@ -15,8 +15,6 @@ source_set("mac") {
   deps = [
     ":dmg_common",
     "//base",
-    "//chrome/common/safe_browsing:archive_analyzer_results",
-    "//chrome/common/safe_browsing:binary_feature_extractor",
     "//components/safe_browsing/core/common/proto:csd_proto",
   ]
 }
