--- a/third_party/node/node.py
+++ b/third_party/node/node.py
@@ -17,11 +17,12 @@ def GetBinaryPath():
   if platform.system() == 'Darwin' and platform.machine() == 'arm64':
       return os.path.join(os_path.join(os_path.dirname(__file__), 'mac',
                           'node-darwin-arm64', 'bin', 'node'))
-  return os_path.join(os_path.dirname(__file__), *{
-    'Darwin': ('mac', 'node-darwin-x64', 'bin', 'node'),
-    'Linux': ('linux', 'node-linux-x64', 'bin', 'node'),
-    'Windows': ('win', 'node.exe'),
-  }[platform.system()])
+  return subprocess.check_output(["which", "node"]).strip()
+#  return os_path.join(os_path.dirname(__file__), *{
+#    'Darwin': ('mac', 'node-darwin-x64', 'bin', 'node'),
+#    'Linux': ('linux', 'node-linux-x64', 'bin', 'node'),
+#    'Windows': ('win', 'node.exe'),
+#  }[platform.system()])
 
 
 def RunNode(cmd_parts, stdout=None):
